### DISCUSSION TITLE
Chat app

### DISCUSSION THREAD

**Discussion by Malmis** [valinet/ExplorerPatcher#631] [General]
Why is the chat app hidden when using windows 10 taskbar? Is it possible to make it visible in the windows 10 taskbar?

**Comment by valinet** [comment]
This was never a feature in Windows 10, so the old taskbar does not support it. Everything is possible, as well as this. StartAllBack implemented this, replacing the icon and the action of the mostly useless “People” button. Very neat idea, I might add. Also, located at the right, near the system tray, much better placement. Where “News and interests” would be.

**Comment by Malmis** [comment]
I know, but the main reason for me to use win 10 taskbar is the "Combine when taskbar is full"-feature. But that feature is not available in the win 11 taskbar.
